packages/runtime/src/templates/getHandler.ts

```diff
@@ -86,7 +102,7 @@ const makeHandler = (conf: NextConfig, app, pageRoot, staticManifest: Array<[str
         port,
       },
       {
-        revalidateToken: customContext.odb_refresh_hooks,
+        revalidateToken: customContext?.odb_refresh_hooks,
```

Comment:
not part of actual fix - just for local debugging (probably will remove it as I will be finishing this PR)

Reply:
This was merged separately ( https://github.com/netlify/next-runtime/pull/2030 )